packages/flutter_tools/lib/src/fuchsia/fuchsia_pm.dart

```diff
@@ -104,6 +104,8 @@ class FuchsiaPM {
       repoPath,
       '-l',
       '$host:$port',
+      '-c',
+      '2',
```

Comment:
does this work if we're serving V1 runners currently?

Reply:
This specifies the version of the config.json file that is served by the package server. The package server serves packages which contain blobs and a component manifest just is a blob in a package that the system knows how to translate into something that can run.

The pkgctl tool on the device expects v2 configs by default but it is backwards compatible with v1 configs if you pass the `-f 1` flag when adding the repo as an update source. The flutter tool controls the interaction between the package server which is serving our application and telling pkgctl to resolve the application so we can serve the new version.